tests/GoActive.Modules.Geo.Domain.Tests/ValueObjectsTests/TestTitle.cs

```diff
@@ -0,0 +1,26 @@
+﻿using FluentAssertions;
+
+using GoActive.Modules.Geo.Domain.ValueObjects;
+
+namespace GoActive.Modules.Geo.Domain.Tests.ValueObjectsTests;
+
+public class TestTitle
+{
+    [Theory]
+    [InlineData(" ")]
+    [InlineData("")]
```

Comment:
Would be better to place inline data in order from empty to non-empty. 
E.g.: 
[InlineData("")]
[InlineData(" ")]

This shows test intentions in more clear way: in the first place you check empty data, then - non-empty (but not valid).